src/alire/alire-properties-labeled.ads

```diff
@@ -88,6 +90,16 @@ private
 
    overriding function Image (L : Label) return String is (Utils.To_Mixed_Case (L.Name'Img) & ": " & L.Value);
 
+   overriding function Key (L : Label) return String is
+     (case L.Name is
+         when Author       => "authors",
+         when Comment      => "comment",
+         when Executable   => "executables",
+         when Maintainer   => "maintainers",
+         when Path         => "paths",
+         when Project_File => "project-files",
+         when Website      => "website");
```

Comment:
Looks like this should directly reference `Alire.TOML_Keys`.

Reply:
Indeed!